Slambook/index.js

```diff
@@ -1,18 +1,25 @@
-let form;
-let allSlamArray=[];
-//Write code below to select the form element
-
```

Comment:
The query selector should target the '.form' class with a dot prefix, like this: '.form'